demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -1101,6 +1101,45 @@ def is_mapping_fields_command_exist(self) -> bool:
                     return False
         return True
 
+    def are_commands_in_readme(self) -> bool:
+        """
+        Check if commands from the YML file exist in the README file.
+        Returns:
+            True if all commands exist, else False.
+        """
+        valid = True
+        commands_to_ignore = ['get-mapping-fields']
+        dir_path = os.path.dirname(self.file_path)
+
+        if not os.path.exists(os.path.join(dir_path, 'README.md')):
+            return True
```

Comment:
how many missing READMEs do we have? should we add a warning for a missing readme? something like that this validation will not run if...

Reply:
I think that it is not relevant to this validation, If Im not mistaken we do have a validation that makes sure that a readme exists, if not , need to open a different issue regarding it.